### PR TITLE
Made the pop up for scoring

### PR DESCRIPTION
**Pull request by sebina**
Made the pop-up for the scoring types

### CODE REVIEW

**Comment by jpbernius** [comment]
image
Visually I do like this modal.

image
Not sure if we need this big of a button, or if the (?) Icon would be sufficient on its own.

File: src/main/webapp/app/quiz/edit/drag-and-drop-question/edit-drag-and-drop-question.component.html [hunk 1]
```diff
@@ -53,6 +53,30 @@
                     <option value="ALL_OR_NOTHING">{{'arTeMiSApp.quizExercise.scoringType.all_or_nothing' | translate}}</option>
                     <option value="PROPORTIONAL_WITH_PENALTY">{{'arTeMiSApp.quizExercise.scoringType.proportional_with_penalty' | translate}}</option>
                 </select>
+
+                <ng-template #info let-c="close" let-d="dismiss">
```
Comment (jpbernius) [inline src/main/webapp/app/quiz/edit/drag-and-drop-question/edit-drag-and-drop-question.component.html]: Am I correct, that the two modals are identical?
What about moving it to its own component instead?

What about creating a `<jhi-quit-scoring-info-dialog>` Component that can be used in Both the Edit Drag and Drop Question Component and the Edit Multiple Choice Question Component.

This Angular Component would display the button and trigger the modal. Reducing the dependencies on `NgbModal` and keeping the Changes to the other components to one line in the HTML Files.

**Comment by krusche** [comment]
I agree, the ? button could be a smaller and less highlighted. Please also change the following wording:
1) right --> correct
2) distract --> subtract
3) one mistake --> a mistake
4) not point at all --> 0 points

At the end of Proportional with Penalty add something like "each wrong answer subtracts 0.5 points."

File: src/main/webapp/app/quiz/edit/drag-and-drop-question/edit-drag-and-drop-question.component.html [hunk 2]
```diff
@@ -53,6 +53,30 @@
...
+                                <th>Proportional with Penalty</th>
+                                <td>Each right answer provides the participant with a fraction of the total score. If one mistake is made, the fraction is distracted from the achieved points.
+                                    For example the total score is 2 and there are 4 options. Each right answers brings the participant 0,5 points </td>
+                        </table>
+                    </div>
+                    <div class="modal-footer">
+                        <button type="button" class="btn btn-outline" (click)="c()">Close</button>
+                    </div>
+                </ng-template>
+                <div><button class="btn btn-info" (click)="open(info)"><i class="fa fa-question-circle-o"></i>&nbsp;Info</button></div>
```
Comment (jpbernius) [inline src/main/webapp/app/quiz/edit/drag-and-drop-question/edit-drag-and-drop-question.component.html]:
```suggestion
                <i class="fa fa-question-circle-o" (click)="open(info)"></i>
```

**Pull request merged**